pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1904,8 +1904,10 @@ ASTUpdateSetClause UpdateSetClause() :
     <SET>
     (
         <ROW> "=" ID()
+        ( "," Column() "=" ( Expression() | <_DEFAULT> )*)*
```

Comment:
Is this correct? I didn't find a clear description, whether a ROW can be combined with single columns at the same time. The ROW uses a record, which contains all columns anyway...

So: Is this actually valid syntax?

```
UPDATE schedule
  SET ROW = default_week,
    Tue = 'Day Off'
  WHERE week = 1;
```